cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -73,6 +76,11 @@ object ReporterCommand : CommandWithHelp() {
             order = PARAMETER_ORDER_MANDATORY)
     private lateinit var reportFormats: List<Reporter>
 
+    @Parameter(description = "A JSON or YAML file that contains resolutions.",
```

Comment:
We don't say which file formats we support in the other parameters, so I guess we shouldn't here either. But instead we should probably mention that this is about *error* resolutions, like "A file containing error resolutions".

Reply:
Done.